src/utils/datetime_utils.js

```diff
@@ -0,0 +1,10 @@
+export function getDateTime(timestamp) {
+	try {
+		const d = new Date(timestamp);
+		const date = d.toLocaleDateString('en-US');
+		const time = d.toLocaleTimeString('en-US', { hour: '2-digit', minute: '2-digit' });
+		return `${date}, ${time}`;
```

Comment:
We shouldn't be returning a string with a specific format like `${date}, ${time}` hardcoded in.
Consider the following use cases:
- What if another component wants to use this format, but another component wants to display time like `${date} - ${time}` instead?
- Or what if a component doesn't care about the time, and only cares about the date?

This function should be able to handle either regardless. To fix this, we should instead return an object that maps keys to values so that each component can use the data however it wants. Like so:

```js
return {
  date: date,
   time: time
}
```

And conveniently, javascript lets a simplify this code into the following that does the exact same thing:
```js
return { date, time }
```

Then, in your components, you can use it like so:

```
const datetime = getDateTime(timestamp);
...

<p> Posted on {datetime.date}, {datetime.time} PST </p>
```